demo/app.py

```diff
@@ -35,8 +67,52 @@ def index():
         response['code'] = res.status_code
         response['data'] = json.loads(res.text)['data']
 
-    return render_template('index.html', form=onid_form, onid=onid, res=response)
+    return render_template('index.html', form=onid_form, onid=onid, res=response, logout_url=logout_url)
+
+@app.route('/logout', methods=['GET'])
+def logout():
+    if 'user' in session:
+        del session['user']
+    return flask.redirect(app.config['CAS_URL'] + "/logout")
```

Comment:
Since you've used `app.config['CAS_URL'] + "/logout"` multiple times in your code, can we use a variable like `CAS_LOGIN_URL` to make it easier to read?

Reply:
I did have a variable for logout url, and I only used the login url once so I didn't create a variable for it. Either way, I'm going to change the logout url to be relative, and reference the variable in the `app.Route()` decorator.

Reply:
Sounds good.